File: bundleparsing2/app/src/main/java/com/example/bundleparsing/MainActivity2.java
package com.example.bundleparsing;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity2 extends AppCompatActivity {
    TextView txt;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
        txt=findViewById(R.id.txt);
        Intent fromact=getIntent();
        String title=fromact.getStringExtra("title");
        String name=fromact.getStringExtra("name");
        int clas=fromact.getIntExtra("class",0);
        txt.setText(name+clas);


        getSupportActionBar().setTitle(title);




    }
}

File: Foodorderingappadminpanel/app/src/main/java/com/example/foodorderingappadminpanel/MainActivity2.kt
package com.example.foodorderingappadminpanel

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.foodorderingappadminpanel.databinding.ActivityMain2Binding


class MainActivity2 : AppCompatActivity() {
    private val binding by lazy {
        ActivityMain2Binding.inflate(layoutInflater)
    }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)
//        binding.button.setOnClickListener {
//            startActivity(Intent(this, ::class.java))
//        }

    }
}